libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-faq-shell',
+  styles: [
+    `
+      :host {
+        display: block;
+        width: calc(200 * var(--watt-space-xs));
```

Comment:
suggestion: Say it like it is.
```suggestion
        width: 800px; // Magic number by designer
```